zkevm-circuits/src/witness/tx.rs

```diff
@@ -691,10 +870,23 @@ impl Transaction {
                     bytes_rlc,
                     gas_cost_acc,
                 },
+                rlp_decoding_table: RlpDecodingTable {
+                    is_write: stack_op.is_write,
+                    id: stack_op.id,
+                    depth: stack_op.depth,
+                    value: stack_op.value,
+                    value_prev: stack_op.value_prev,
+                    stack_acc: stack_op.stack_acc,
+                    stack_acc_pow_of_rand: stack_acc_pow_of_rand[stack_op.depth],
+                    is_stack_init: stack_op.is_init,
+                    is_stack_push: stack_op.is_push,
+                    is_stack_pop: stack_op.is_pop,
+                    is_stack_update: stack_op.is_update,
+                },
```

Comment:
minor refactoring request. Again for better maintenance of codebase. I would suggest:

```suggestion
                rlp_decoding_table: stack_op.into(),
```

or something similar like `RlpDecodingTable::from_stack_op(pow_of_rand)`

Reply:
`RlpDecodingTable` is removed.